### PR TITLE
Update development document with more information.

### PR DESCRIPTION
**Pull request by balamurugana**
Signed-off-by: Bala.FA <bala.gluster@gmail.com>

### CODE REVIEW

File: docs/development-and-testing.md [hunk 1]
```diff
@@ -1,34 +1,56 @@
 ## Development and Testing
-0. You would need to have `quay.io` account, running docker and kubernetes; and make sure `quay.io/<QUAY_USERNAME>/direct-csi` repository is `public`.
-1. Go to project root
+1. You would need to have `quay.io` account, running docker and kubernetes.
```
Comment (wlan0) [inline docs/development-and-testing.md:2]: This is not a requirement. registry can be configured using `--registry`

File: docs/development-and-testing.md [hunk 2]
```diff
@@ -1,34 +1,56 @@
...
+2. Docker login to your `quay.io` account.
```
Comment (wlan0) [inline docs/development-and-testing.md:3]: Should instead be

"Login to your registry account using the following command. For instance, if your registry is quay.io..."

File: docs/development-and-testing.md [hunk 3]
```diff
@@ -1,34 +1,56 @@
...
+```bash
+$ docker login --username <QUAY_USERNAME> https://quay.io
+```
+3. If you do not have `csi-provisioner`, `livenessprobe` or `csi-node-driver-registrar`, create those repositories by pull/tag/push to your `quay.io` account.
```
Comment (wlan0) [inline docs/development-and-testing.md:7]: This operation is idempotent. We should just say

Creates the following images in your registry account:

1. ..
2. ..
3. ..

File: docs/development-and-testing.md [hunk 4]
```diff
@@ -1,34 +1,56 @@
...
+
+$ docker pull quay.io/minio/livenessprobe@sha256:6f056a175ff4ead772edc9bf99aef74c275a83c51868dd26090dcb623425a742 && \
+docker tag ef2b13b2a066 quay.io/<QUAY_USERNAME>/csi-node-driver-registrar && \
+docker push quay.io/<QUAY_USERNAME>/csi-node-driver-registrar
+   
+$ docker pull  quay.io/minio/csi-node-driver-registrar@sha256:9f9ce5c98e44d66b8ad34351616fdf78765b9f24c3c3b496cee784dadf63f528 && \
+docker tag de977053da40 quay.io/<QUAY_USERNAME>/livenessprobe && \
+docker push quay.io/<QUAY_USERNAME>/livenessprobe
+```
+4. Make sure `csi-provisioner`, `livenessprobe` or `csi-node-driver-registrar` repositories are `public` in your `quay.io` account.
```
Comment (wlan0) [inline docs/development-and-testing.md:21]: 👍🏾

File: docs/development-and-testing.md (hunk 1 above)
Comment (balamurugana) [reply docs/development-and-testing.md]: Fixed in different way.

File: docs/development-and-testing.md (hunk 2 above)
Comment (balamurugana) [reply docs/development-and-testing.md:3]: Fixed in different way.

File: docs/development-and-testing.md (hunk 3 above)
Comment (balamurugana) [reply docs/development-and-testing.md]: Fixed in different way.

**Review by wlan0 — APPROVED** [review]

**Pull request merged**